src/experiments.js

```diff
@@ -121,3 +138,13 @@ function saveExperimentIds(win, experimentIds) {
   setCookie(win, COOKIE_NAME, experimentIds.join(','),
       timer.now() + COOKIE_EXPIRATION_INTERVAL);
 }
+
+/**
+ * Resets the experimentsToggle cache for testing purposes.
+ * @visibleForTesting
+ */
+export function resetExperimentToggles_() {
```

Comment:
it already is ran in `afterEach`. i was lazy and used the same `e1` value. ill go ahead and change them.